documents/resources.py

```diff
@@ -0,0 +1,25 @@
+import documents.state_machine as state_machine
+import documents.decorators as decorators
+from flask import jsonify, request, Blueprint
+
+documents = Blueprint("documents", __name__, template_folder="templates")
+
+@documents.route('', methods=['POST'])
+@decorators.utils
+@decorators.auth
+def upsert_documents(payload):
+    user_id = payload['userid']
+    state_machine.get_user_by_user_id(user_id)
+    document_id = state_machine.upsert_document(user_id)
```

Comment:
why isn't upsert document returning the entire object? it shouldn't be returning just the id